iphone/cli/commands/_build.js

```diff
@@ -4500,6 +4502,8 @@ iOSBuilder.prototype.writeXcodeConfigFiles = function writeXcodeConfigFiles() {
 			'OTHER_LDFLAGS[sdk=iphonesimulator*]=$(inherited) $(JSCORE_LD_FLAGS)',
 			'OTHER_LDFLAGS[sdk=iphoneos9.*]=$(inherited) -weak_framework Contacts -weak_framework ContactsUI -weak_framework WatchConnectivity -weak_framework CoreSpotlight',
 			'OTHER_LDFLAGS[sdk=iphonesimulator9.*]=$(inherited) -weak_framework Contacts -weak_framework ContactsUI -weak_framework WatchConnectivity -weak_framework CoreSpotlight',
+			'GCC_DEFINITIONS=' + this.gccDefs,
```

Comment:
Again, here's where I'd turn the `Map` to a `string`, basically just joining key/value pairs with `'='` and entries with `' '` as a naive approach. (I assume we may eventually need to quote some values if they have spaces or something).